src/components/editor/PropertyTemplateOutline.jsx

```diff
@@ -82,8 +82,13 @@ export class PropertyTemplateOutline extends Component {
     })
   }
 
+  /*
+   * This is fired when the expand/collapse button is clicked
+   */
   addPropertyTypeRows = (property) => {
-    let existingJsx; let
+    let existingJsx
+
+    let
```

Comment:
ditto ☝️

Reply:
This stuff was from doing `eslint --fix`. I don't know why it fixed it this way.  Bug in eslint?